test/crdsvalidation/konnect.konghq.com/konnectgatewaycontrolplane_test.go

```diff
@@ -116,6 +116,8 @@ func TestKonnectGatewayControlPlane(t *testing.T) {
 		common.TestCasesGroup[*konnectv1alpha1.KonnectGatewayControlPlane]{
 			{
 				Name: "konnect.authRef change is not allowed for Programmed=True",
+				// Tracking issue: https://github.com/Kong/kubernetes-configuration/issues/504
+				SkipReason: "Fails when v1alpha2 is the storage version due to missing conversion logic. This test could be re-enabled by implementing conversion logic using a conversion webhook",
```

Comment:
Nit: Can we move the issue link to the skip message? This way it shows up in test logs.